ansible/main.yml

```diff
@@ -40,8 +40,6 @@
       tags: "websites"
     - role: "digidecs"
       tags: "digidecs"
-    # - role: "freight"  Commented out, as this seems to be unused and is incompatible with Ubuntu 22.04
```

Comment:
this comment can now go

Reply:
Silly me, only removed the tags line.

Reply:
Wait, I am literally making the comment go away in the latest commit. Perhaps I am misunderstanding you, but line 43 and 44 are now gone.

Reply:
wow you're right, brainfart